apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -406,7 +430,11 @@ defmodule AndiWeb.EditLiveView do
     socket = reset_save_success(socket)
 
     dataset = Datasets.get(socket.assigns.dataset.id)
-    changeset = InputConverter.andi_dataset_to_full_ui_changeset(dataset)
+
+    changeset =
+      dataset
+      |> InputConverter.andi_dataset_to_full_ui_changeset()
+      |> Map.put(:action, :update)
```

Comment:
I'd love to see us DRY up some of this weirdness with `:action` I think it's also in `finalize_validation`, unless that has changed